src/components/organisms/AdminVenueView/AdminVenueView.tsx

```diff
@@ -159,6 +158,7 @@ export const AdminVenueView: React.FC<AdminVenueViewProps> = ({
               to={visitSpaceUrl}
               name="Visit space"
               variant="primary"
+              target="_blank"
```

Comment:
I don't know if this is safe. Will the link have `rel="noreferrer noopener"`? Is it needed?

Reply:
thanks! Fixed: https://github.com/sparkletown/sparkle/pull/2969/commits/54d690c68fa6b074181f6d18a88fb443a9797d0d